header/p2p/pb/extended_header_request.proto

```diff
@@ -9,3 +9,8 @@ message ExtendedHeaderRequest {
   }
   uint64 amount = 3;
 }
+
+message ExtendedHeaderResponse {
+  bytes body = 1;
+  uint32 statusCode = 2;
```

Comment:
We should make statusCode as enum defined in proto. This way, the protocol definition would be self-sufficient. The convertStatusCodeToError can just rely on the PB definition